parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -17,14 +17,12 @@ else
   export GPU_DV=418.40.04
 fi
 export GPU_DEST=/usr/local/nvidia
-NVIDIA_DOCKER_VERSION=2.0.3
-DOCKER_VERSION=1.13.1-1
-NVIDIA_CONTAINER_RUNTIME_VER='3.5.0*'
 PRIVATE_IP=$( (ip -br -4 addr show eth0 || ip -br -4 addr show azure0) | grep -Po '\d+\.\d+\.\d+\.\d+')
 if ! [[ $(echo -n "$PRIVATE_IP" | grep -c '^') == 1 ]]; then
   PRIVATE_IP=$(hostname -i)
 fi
 export PRIVATE_IP
+APT_CACHE_DIR=/var/cache/apt/archives/
```

Comment:
Not for now, but at some point we can look at cleaning up the cache after the node has fully provisioned.  (Just to free up space for containers/etc)

Reply:
Makes sense, and I think more generally the apt cache can probably be cleaned up after the bootstrap script finishes (probably in the background, non-blocking)